lib/timetable-form/Form.tsx

```diff
@@ -169,6 +221,13 @@ export const Form = (props: React.HTMLAttributes<HTMLDivElement>) => {
           <Button>Export Timetable Header CSV</Button>
         </a>
       </div>
+      <Button
+        onClick={() => {
+          handleValidateForm();
+        }}
```

Comment:
You could just hand this directly: `onClick={handleValidateForm}` (otherwise we're getting no benefit out of using `useCallback`)

Reply:
I will clean this up